### ISSUE TITLE
Figure out how to assign new Takeout users to the correct channel for the GH:E version

### ISSUE THREAD

**Issue by burtonjc** [waffleio/waffle.io#3617]
Let's just make the GHE-2.11 channel the default channel and then the manual step would be to move qualified customers up to the latest Takeout. This way, in theory, no one would ever be broken, but they might install and then have a bunch of upgrades. Maybe we should test this process to make sure it works.

requires https://github.com/waffleio/hidden/issues/370

**Comment by Alisuehobbs** [comment]
I set unstable to the `Stable-ghe-2.11` channel and it installed version 2.9.5(699) correctly. I then moved it over to the `Stable` channel, clicked `Sync License` and installed the 2 upgrades on that channel. It took about 10 minutes and only required two clicks. The question now is, is this easy enough so that we can set all new installations to the `Stable-ghe-2.11` channel and then move them over to other channels as appropriate?
